plugins/vpc-branch-eni/plugin/commands.go

```diff
@@ -256,6 +256,7 @@ func (plugin *Plugin) Del(args *cniSkel.CmdArgs) error {
 
 			return nil
 		})
+		log.Errorf("Failed to set netns %s, ignoring %v", args.Netns, err)
```

Comment:
Do we want to log even when err==nil?
Also, do we want to return the error from this function?